### ISSUE TITLE
Стеження за темою (нові повідомлення / оновлення роздачі)

### ISSUE THREAD

**Issue by chief-j** [hurtom/toloka#65]
Пропоную зробити сповіщення про оновлення в темі (щось подібне до дзвіночка на github).
У темі розмістити кнопку "Стежити", можна з опціями "Нові повідомлення", "Оновлення роздачі (Якщо це роздача)".
Бажано розділити приватні повідомлення із сповіщеннями, для зручності

**Comment by yukoff** [comment]
Так, я думав про це коли малював ескізи, але поки не додавав. Тут нема що розділяти - сповіщення ж про всі події, просто в списку подій буде список оновлень в темах, та приватних повідомлень (ну, це якщо вже взітий ґітхаб для приду). Я ще думав про показ push-сповіщень - але це на потім.

**Comment by chief-j** [comment]
Все таки, треба розділити. Приватні Повідомлення для спілкування, а не для сповіщень. Коли нікого на роздачі немає, ти кличеш сідерів. Від тебе відправляється сотня повідомлень, це зайве загромадження, коли можна зробити звичайне сповіщення.
До речі, треба ще додати сповіщення "Треба сідера" (або щось у цьому дусі)

**Comment by yukoff** [comment]
@chief-j Тут все до купи. Сповіщення - це про нові події. Нове приватне повідомлення - подія. Оновлення в треді - подія. Кличуть сідера - подія. Тобто "по дзвіночку" буде видно три типи подій.

**Comment by chief-j** [comment]
@yukoff ти все ускладнюєш

**Comment by yukoff** [comment]
А як на мене - куди вже простіше. Може ми просто про різні речі?

**Comment by chief-j** [comment]
https://toloka.to/t9030-1770#882085
> Організуйте, будь-ласка, на сайті підписку на оновлення торенту. Це дуже зручно для користувачів. Вважаю, що цього хотів би не я один. І ніхто не пропустиь ніяких оновлень﻿.

**Comment by yukoff** [comment]
https://toloka.to/t81976-90#924123
> не знаю чи про це писали, але чи є можливість коли сайт відкритий аби були сповіщення про повідомлення візуальні або хоча би звукові, було б непогано

**Comment by yukoff** [comment]
<https://toloka.to/t9030-1920#924728>
> Також така дрібниця, як повідомлення на пошту про приватні повідомлення, хотілось би бачити нік людини в повідомленні на пошту.
